src/Analysis/Ast/Impl/Types/Definitions/IArgumentSet.cs

```diff
@@ -141,5 +141,10 @@ public interface IArgumentSet {
         /// Evaluator associated with the set.
         /// </summary>
         IExpressionEvaluator Eval { get; }
+        
+        /// <summary>
+        /// Expression in how argument set was called
```

Comment:
`Call expression that invoked function or method with this argument set`

Reply:
I agree, mine was pretty informal.